### ISSUE TITLE
Installer fails to retrieve zim files

### ISSUE THREAD

**Issue by Popolechien** [offspot/cardshop#283]
https://cardshop.hotspot.kiwix.org/orders/L755R61B7, https://cardshop.hotspot.kiwix.org/orders/L754R61B7, and  https://cardshop.hotspot.kiwix.org/orders/L753R61B7 all failed with a variation of `Error downloading http://mirror.download.kiwix.org/zim/nameofzimfile.zim to /data/kiwix-hotspot.cache/nameofzimfile.zim
aria2c returned 1`
With said zim files being ' wikipedia_en_all_maxi' or ' cest-pas-sorcier_fr_all'

**Comment by Popolechien** [comment]
Ok I stand corrected: the installer does not _fail_ to retrieve the zim file: it is more like the connection breaks at some point.

In my latest run ([L757R61B8](https://cardshop.hotspot.kiwix.org/orders/L757R61B8)) the installation crashed while it was 16% into the download of `cest-pas-sorcier_fr_all`

**Comment by rgaudin** [comment]
Was already the case for the other ones ; first one failed after download 48GiB for instance

**Comment by Popolechien** [comment]
Yes yes but I had written a generic "fails", I wanted to put it on record in the ticket what the issue was.

**Comment by Popolechien** [comment]
Both https://cardshop.hotspot.kiwix.org/orders/L765R61C0 and https://cardshop.hotspot.kiwix.org/orders/L766R61C0 crashed again, again with C'est pas sorcier, around 15 and 23% of it. I did have another config in English with `Wikipedia_en_all_maxi` go through, though.

**Comment by Popolechien** [comment]
Created [a new order](https://cardshop.hotspot.kiwix.org/all-orders/61d60faa98c8bf5804647993) with the same config from which I only removed the usual point of failure (`C'est pas sorcier`) and relaunched: it failed again with error being on KAlite (not a zim therefore):
```
Error downloading http://mirror.download.kiwix.org/other/kalite/videos_fr_0.17.tar to /data/kiwix-hotspot.cache/kalite_videos_fr.tar
aria2c returned 1
```

**Comment by rgaudin** [comment]
OK, it's the first time it's not happening on a ZIM AFAIK. It's also a much smaller file (10GB)

**Comment by Popolechien** [comment]
Re-added C'est pas sorcier, which is now last on the list: after a couple of recreations [(61d84cba98c8bfb26d64a327](https://cardshop.hotspot.kiwix.org/all-orders/61d84cba98c8bfb26d64a327), [61d840a498c8bf4811649001](https://cardshop.hotspot.kiwix.org/all-orders/61d840a498c8bf4811649001)) it clearly is not the cause of this issue as the creation fails now consistently on the same yet different 15G file (at 44 and 73%):
```
Error downloading http://mirror.download.kiwix.org/zim/other/nota-bene_fr_all_2020-04.zim to /data/kiwix-hotspot.cache/nota-bene_fr_all.fr.zim
aria2c returned 1
```

**Comment by Popolechien** [comment]
Adding [L819R61E7](https://cardshop.hotspot.kiwix.org/orders/L819R61E7) and [L818R61E7](https://cardshop.hotspot.kiwix.org/orders/L818R61E7)

First one crashed on c'est pas sorcier, the second one on KAlite videos (exact same as above `videos_fr_0.17.tar`)

**Comment by Popolechien** [comment]
Adding [L827R61ED](https://cardshop.hotspot.kiwix.org/orders/L827R61ED) which failed on 
```
Error downloading http://mirror.download.kiwix.org/zim/wikihow/wikihow_en_maxi_2022-01.zim to /data/kiwix-hotspot.cache/wikihow_en_maxi.en.zim
aria2c returned 1
```
There have been three or four successful image creations with similar setups (`c'est pas sorcier`) before that so clearly this is more of something acting up that a clear bug.

**Comment by kelson42** [comment]
@rgaudin Time to reassess/fix this issue considering the new imager downloader is going to be harden.
